### PR TITLE
[WIP] backup/import private keys

### LINKED ISSUE #2680: backup private keys
Settings page, show Backup private keys option that would potentially allow people to display and copy private key or password stored on device, only show for standard logins, non hivesigner login.

### PR DESCRIPTION
**Pull request by aliseyalvi**
### What does this PR?
This PR adds functionality of viewing/importing private/public keys stored in the app.
User can view/copy private or public keys and if user is logged in with active/posting/memo key he can import other keys by providing master key / password

### Issue number
fixes #2680 

### Screenshots/Video
https://discord.com/channels/@me/920267778190086205/1103758928313077853

### CODE REVIEW

**Review by noumantahir — CHANGES_REQUESTED** [review]
All seems to be good, require review from @feruzm, I added one comment as review regarding import keys when using hiveSigner, do check that .... 

other than that, one suggestion I may add, perhaps we should ask for pinCode if it's active before letting user access keys screen, it should be a simple step to integrate....
Screenshot 2023-05-22 at 5 40 55 PM
Screenshot 2023-05-22 at 5 40 35 PM

File: src/screens/settings/container/settingsContainer.tsx
Comment (noumantahir) [inline src/screens/settings/container/settingsContainer.tsx:308]: @aliseyalvi please redo this part, the current implementation will,

- make users not using pinCode unable to access manage screen
- break functionality for those already using pinCode as your configuration for pinCode screen will put it in reset mode...

for reference check code here: https://github.com/ecency/ecency-mobile/blob/a93b36b1d1705e4b58211a33319d78978461a2ea/src/screens/assetDetails/screen/assetDetailsScreen.tsx#L147

**Review by noumantahir — CHANGES_REQUESTED** [review]
@aliseyalvi please carefully test your implementations, the pinCode integration you have done have several issues I have added in review comment, though I do understand this might be your first time working with pinCode screen......

**Review by noumantahir — COMMENTED** [review]
Seems good from working perspective, @feruzm as for flow of things, since I was not directly part of discussion, perhaps you should review that part... if all good, can be approved and merged.

demo video:
https://github.com/ecency/ecency-mobile/assets/6298342/2e963070-6a47-44ab-9c26-34804693d0b2

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**
